frontend/src/components/global/gameplaySidePanel/MoveListPanel.tsx

```
@@ -1,83 +1,83 @@
+            return "0-1"
+        }
+    }
+
+    return (
+        <div className="move-list-panel-container">
+            <h4 className="move-list-header">Moves</h4>
+            <div className="moves-container">
+                {moveList.map(
+                    (movePair: Array<string>, movePairIndex: number) => {
+                        return (
+                            <div className="move-info-container">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)